go.mod

```diff
@@ -8,6 +8,9 @@ replace (
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200121204235-bf4fb3bd569c
 )
 
+// TODO: Remove this when https://github.com/vishvananda/netlink/pull/554 is merged
```

Comment:
You can comment go.mod??? TIL, thx 🤯

Reply:
It ought to be possible, `go.mod` files seen to just be regular go sources with some special sauce builtins.